schemas/2019-04-01/deploymentTemplate.json

```diff
@@ -968,6 +968,10 @@
                   { "$ref": "https://schema.management.azure.com/schemas/2017-08-15/Microsoft.NetApp.json#/resourceDefinitions/capacityPools" },
                   { "$ref": "https://schema.management.azure.com/schemas/2017-08-15/Microsoft.NetApp.json#/resourceDefinitions/volumes" },
                   { "$ref": "https://schema.management.azure.com/schemas/2017-08-15/Microsoft.NetApp.json#/resourceDefinitions/snapshots" },
+                  { "$ref": "https://schema.management.azure.com/schemas/2019-05-01/Microsoft.NetApp.json#/resourceDefinitions/netAppAccounts" },
+                  { "$ref": "https://schema.management.azure.com/schemas/2019-05-01/Microsoft.NetApp.json#/resourceDefinitions/capacityPools" },
+                  { "$ref": "https://schema.management.azure.com/schemas/2019-05-01/Microsoft.NetApp.json#/resourceDefinitions/volumes" },
+                  { "$ref": "https://schema.management.azure.com/schemas/2019-05-01/Microsoft.NetApp.json#/resourceDefinitions/snapshots" },
```

Comment:
>snapshots [](start = 127, length = 9)

netAppAccounts_capacityPools_volumes_snapshots

Reply:
> > snapshots [](start = 127, length = 9)
> 
> netAppAccounts_capacityPools_volumes_snapshots

Yes of course. These were renamed in the latest version which was auto generated. Will fix.